Dockerfile

```
@@ -1,6 +1,6 @@
-FROM node:20-alpine3.17
+FROM node:23-alpine3.20
 
-RUN apk add --no-cache python3 make g++  \
+RUN apk add --no-cache python3 make g++ curl \
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)